README.md

```diff
@@ -906,11 +907,54 @@ internal class Settings
 Once we define our settings we can access them using the `Global` property inside `Instance`.
 ```csharp
 //Get user from settings
-var user = SettingsProvider<Settings>.Instance.Global.User
+var user = SettingsProvider<Settings>.Instance.Global.User;
 
  //Modify the port 
  SettingsProvider<Settings>.Instance.Global.Port = 20;
  
  //if we want those settings to persist
  SettingsProvider<Settings>.Instance.PersistGlobalSettings();
 ```
+
+### The `Connection` class
+It represents a network connection with 2 operation modes (server or client side). It wraps a TcpClient and its corresponding network streams.
+
+[Connection API Doc](https://unosquare.github.io/swan/api/Unosquare.Swan.Networking.Connection.html)
+
+[ConnectionListener API Doc](https://unosquare.github.io/swan/api/Unosquare.Swan.Networking.ConnectionListener.html)
+
+#### Example 1: Creating a connection on the server side
```

Comment:
Change title to "Creating an echo server" and include logic to read from connection and response the same message